RDF.js

```diff
@@ -1135,8 +1144,9 @@ function doImport() {
 	// keep track of collections while we're looping through
 	var collections = new Array();
 	
-	var i = 0;
-	for each(var node in nodes) {
+	var j = 0;
+	for (var j=0; i<nodes.length; j++) {
+		var node = nodes[j];
```

Comment:
This is really bizarre logic (also slightly off, since `i` doesn't start from 0 necessarily). Just use `I` in the for loop and drop the additional declaration above. Also fix the progress tracker below to be `i+1` (it seems to have been wrong, though it's still not quite right)